.github/actions/label-and-move-released-issues-v1/src/run.test.ts

```diff
@@ -270,7 +254,26 @@ describe('run', () => {
             repo: 'repo'
           }
         },
-        getOctokit: () => octokit
+        getOctokit: () => {
```

Comment:
As of `@actons/github` v6, they changed the way the generate the GET endpoints which completely broke the way we were calling them, they appear to be proxying old + new endpoints with a new `handler` function which attempted to destructure a variable which ended being undefined: 

>       TypeError: Cannot destructure property ‘decorations’ of ‘endpointMethodsMap.get(...).get(...)’ as it is undefined.



Similar, not exact issue: https://github.com/octokit/plugin-rest-endpoint-methods.js/issues/668


I'll be opening a issue in that repo, but for now this a work around

Reply:
Opened an issue to see if this was an intended change as of v6: https://github.com/octokit/plugin-rest-endpoint-methods.js/issues/683

Reply:
This issue was fixed by GitHub but the fix needs to be upstreamed to `@actions/github` before I can pull in the change. Will wait for that before merging, moving this to draft.

Reply:
I was waiting for the above PR to be merged but it's not moving. 


Will un-draft this PR and get this moving, opened issue to revert the work around once there has been some movement: https://github.com/dequelabs/axe-api-team-public/issues/80